From 91d8e6d83de602c3fb4b433b3f8197fc052564ff Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Tue, 28 Jul 2020 09:14:08 +0800
Subject: [PATCH] Test case for included DebugException.h changing - nasm file

Description: Change the DebugException.h files which included in the SourceLevelDebugPkg\Library\DebugAgent\DebugAgentCommon\Ia32\AsmFuncs.nasm.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 .../Library/DebugAgent/DebugAgentCommon/Ia32/DebugException.h   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SourceLevelDebugPkg/Library/DebugAgent/DebugAgentCommon/Ia32/DebugException.h b/SourceLevelDebugPkg/Library/DebugAgent/DebugAgentCommon/Ia32/DebugException.h
index 9147fabe6c..f57fc6e660 100644
--- a/SourceLevelDebugPkg/Library/DebugAgent/DebugAgentCommon/Ia32/DebugException.h
+++ b/SourceLevelDebugPkg/Library/DebugAgent/DebugAgentCommon/Ia32/DebugException.h
@@ -15,7 +15,7 @@
 #define DEBUG_EXCEPT_BREAKPOINT               3
 #define DEBUG_EXCEPT_OVERFLOW                 4
 #define DEBUG_EXCEPT_BOUND                    5
-#define DEBUG_EXCEPT_INVALID_OPCODE           6
+#define DEBUG_EXCEPT_INVALID_OPCODE           7
 #define DEBUG_EXCEPT_DOUBLE_FAULT             8
 #define DEBUG_EXCEPT_INVALID_TSS             10
 #define DEBUG_EXCEPT_SEG_NOT_PRESENT         11
-- 
2.27.0.windows.1

